CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 - **String**:
   - Overloaded Swift's 'contains' operator (`~=`) for `String` to check regex matching. [#858](https://github.com/SwifterSwift/SwifterSwift/pull/858) by [VatoKo](https://github.com/VatoKo)
   - `regexEscaped`, which returns an escaped string for inclusion in a regex pattern
+  - Added `matches(regex:options:)` and the `~= regex` to check directly against NSRegularExpression. Also added `replacingOccurrences(pattern:template:searchRange:)` and `replacingOccurrences(regex:template:searchRange:)` as a more convenient way to replace patterns and NSRegularExpressions. 
```

Comment:
If you take my suggested change, you'll also need to move this to `StringProtocol.
```suggestion
  - Added `matches(regex:options:)` and the `~= regex` to check directly against NSRegularExpression. Also added `replacingOccurrences(ofPattern:withTemplate:options:searchRange:)` and `replacingOccurrences(regex:template:searchRange:)` as a more convenient way to replace patterns and NSRegularExpressions. 
```